<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>679</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Произношение</string>
  </property>
  <widget class="QLineEdit" name="lineEditInput">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>70</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelInput">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Английское слово  (Enter) :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOutput">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>35</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Phonon::VolumeSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>186</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="maximumVolume">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>600</y>
     <width>191</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Добавить файл</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonRepeat">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>291</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string> Повторять (Alt + ←)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>125</y>
     <width>66</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetFiles">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>306</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>171</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Звуковые файлы:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Вывести словосочетания (Alt + →)</string>
   </property>
  </widget>
  <widget class="QListWidget" name="cacheWord">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>175</y>
     <width>241</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::NoDragDrop</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>135</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Кеш:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>135</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Перевод:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>175</y>
     <width>306</width>
     <height>471</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditInput</sender>
   <signal>textChanged(QString)</signal>
   <receiver>labelOutput</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonRepeat</sender>
   <signal>clicked()</signal>
   <receiver>lineEditInput</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lineEditInput</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditInput</sender>
   <signal>returnPressed()</signal>
   <receiver>Form</receiver>
   <slot>pressedEnter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandLinkButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>choiceFileInExplorer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidgetFiles</sender>
   <signal>itemActivated(QListWidgetItem*)</signal>
   <receiver>Form</receiver>
   <slot>choiceItemFromList(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidgetFiles</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>Form</receiver>
   <slot>choiceItemFromList(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>lineEditInput</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cacheWord</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>Form</receiver>
   <slot>choiceItemFromCacheWord(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cacheWord</sender>
   <signal>itemActivated(QListWidgetItem*)</signal>
   <receiver>Form</receiver>
   <slot>choiceItemFromCacheWord(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>707</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>709</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setFocusLineEdit()</slot>
  <slot>pressedEnter()</slot>
  <slot>choiceFileInExplorer()</slot>
  <slot>choiceItemFromList(QListWidgetItem*)</slot>
  <slot>choiceItemFromCacheWord(QListWidgetItem*)</slot>
 </slots>
</ui>
